src/components/tabs/Tabs.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Tab from './Tab';
+
+import '../../../assets/css/tabs.css';
+
+class Tabs extends Component {
+  static propTypes = {
+    children: PropTypes.instanceOf(Array).isRequired,
```

Comment:
Best propTypes for 'children' is:
```
PropTypes.oneOfType([
        PropTypes.arrayOf(PropTypes.node),
        PropTypes.node
    ]).isRequired
```

Reply:
Thank you for pointing it out, i will change it!